en/ui-system/components/engine/ui-material.md

```diff
@@ -0,0 +1,47 @@
+# UI Custom Material
+We supports UI custom material in sprite component since v1.2, the user interface is as follows:
```

Comment:
We supports UI custom material in sprite component since v1.2, the user interface is as follows:
-->
__SpriteComponent__ supports __UI Custom Material__. The user interface is as follows: